src/test/java/picard/analysis/artifacts/ConvertSequencingArtifactToOxoGTest.java

```diff
@@ -60,74 +77,78 @@ public void testEquivalence() throws IOException, NoSuchFieldException {
         args.add("REFERENCE_SEQUENCE=" + REFERENCE_SEQUENCE.getAbsolutePath());
 
         CollectSequencingArtifactMetrics collectArtifactMetrics = new CollectSequencingArtifactMetrics();
-        Assert.assertEquals(collectArtifactMetrics.instanceMain(args.toArray(new String[args.size()])), 0,
+        Assert.assertEquals(collectArtifactMetrics.instanceMain(args.toArray(new String[0])), 0,
                 "CollectSequencingArtifactMetrics can't process " + SAM_FILE.getAbsolutePath() + " correctly");
 
         args.clear();
-        args.add("OUTPUT_BASE=" + convertedArtifacts.getAbsolutePath());
-        args.add("INPUT_BASE=" + outputFileArtifacts.getAbsolutePath());
-
+        if (newArgStyle) {
+            args.add("OXOG_OUT=" + oxogMetricsOutput);
+            args.add("BAIT_BIAS_IN=" + baitBiasDetails);
+            args.add("PRE_ADAPTER_IN=" + preAdapterDetails);
+        } else {
+            args.add("OUTPUT_BASE=" + convertedArtifacts.getAbsolutePath());
+            args.add("INPUT_BASE=" + outputFileArtifacts.getAbsolutePath());
+        }
         ConvertSequencingArtifactToOxoG convertSequencingArtifactToOxoG = new ConvertSequencingArtifactToOxoG();
-        Assert.assertEquals(convertSequencingArtifactToOxoG.instanceMain(args.toArray(new String[args.size()])), 0,
+        Assert.assertEquals(convertSequencingArtifactToOxoG.instanceMain(args.toArray(new String[0])), 0,
                 "ConvertSequencingArtifactToOxoG can't process base input" + outputFileArtifacts.getAbsolutePath() + " correctly");
 
-        String[] columnToCompare=new String[]{
-                "SAMPLE_ALIAS", "LIBRARY", "CONTEXT", "TOTAL_BASES", "REF_NONOXO_BASES", "REF_OXO_BASES", "REF_TOTAL_BASES", "ALT_NONOXO_BASES", "ALT_OXO_BASES", "OXIDATION_ERROR_RATE", "OXIDATION_Q", "C_REF_REF_BASES", "G_REF_REF_BASES", "C_REF_ALT_BASES", "G_REF_ALT_BASES", "C_REF_OXO_ERROR_RATE", "C_REF_OXO_Q", "G_REF_OXO_ERROR_RATE", "G_REF_OXO_Q"};
-
-        Assert.assertTrue(areMetricsEqual(outputFileOxoG, new File(convertedArtifacts + ".oxog_metrics"), columnToCompare), "Metrics differ");
+        Assert.assertTrue(areMetricsEqual(outputFileOxoG, oxogMetricsOutput, COLUMNS_TO_COMPARE), "Metrics differ");
     }
 
     /**
      * Compare the metrics in two files, ignoring headers and histograms.
      */
     public static boolean areMetricsEqual(final File file1, final File file2, final String[] columnsToCompare) throws NoSuchFieldException {
-        try {
+        try (Reader reader1 = new FileReader(file1);
```

Comment:
final, final

Reply:
They already are! Resources declared in a `try` are `final` by default. https://docs.oracle.com/javase/specs/jls/se8/html/jls-14.html#jls-14.20.3